ext/fuzztest.cmd

```diff
@@ -9,10 +9,12 @@
 
 git clone https://github.com/google/fuzztest.git
 cd fuzztest
-: # There is no tagged release as of 2024/02/23. Pick the last commit that works.
-git checkout b39227cb001a46ed007fa37e40507d777652ede9
+: # There is no tagged release as of 2024/04/09. Pick the latest commit that works.
+git checkout a990537097b6fd337436f7662fb39fc91fa01f3f
```

Comment:
After this line, add a comment to link to https://github.com/google/fuzztest/issues/1124?

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/2096